### PR TITLE
[ADD] open_academy: Creation of course model T#59081

### PR DESCRIPTION
**Pull request by JesusValdez96**
Add course model from exercise 2.
link to exercise: https://www.odoo.com/documentation/15.0/developer/howtos/backend.html#special-fields

### CODE REVIEW

**Comment by andreaarce** [comment]
Please on the PR description add more information about for what is this change and not the description of the exercise.

The same applies to the commit. 

Also to the python files add a new line at the end of the file

**Comment by andreaarce** [comment]
@CarmenMiranda Please review

**Comment by CarmenMiranda** [comment]
Hi @andreaarce I have the same comments that you said already, the new lines at the end in the files, and a better description in the commit, just one other thing is that the name should be `[ADD]` instead of `[IMP]` because all the things that are being done are adding new files

**Comment by JesusValdez96** [comment]
Please review @andreaarce, @CarmenMiranda

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# mqt pre-commit lint check files
+.eslintrc.json
+.flake8
+.pre-commit-config-optional.yaml
+.pre-commit-config.yaml
+.pylintrc
+.pylintrc-optional
```
Comment (CarmenMiranda) [inline .gitignore:7]: Missing new line at the end of the file

**Review by CarmenMiranda — CHANGES_REQUESTED** [review]
Just another new line that is missing, also, can you change the name of the PR with the ADD, and if possible in the commit add a link to the exercise

Also, update the description in the PR with the message that you put in the commit.

**Comment by JesusValdez96** [comment]
Thank you!
Changes made @andreaarce, @CarmenMiranda

**Review by CarmenMiranda — APPROVED** [review]
Technically LGTM 👍🏼  @JesusValdez96 Just remove the draft in the name of the PR and it will be ready to merge @andreaarce

**Pull request merged**